Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -857,49 +880,62 @@ def route_edl() -> Response:
     username: str = credentials.get('identifier', '')
     password: str = credentials.get('password', '')
     cache_refresh_rate: str = params.get('cache_refresh_rate')
+
     if username and password:
         headers: dict = cast(Dict[Any, Any], request.headers)
         if not validate_basic_authentication(headers, username, password):
             err_msg: str = 'Basic authentication failed. Make sure you are using the right credentials.'
             demisto.debug(err_msg)
-            return Response(err_msg, status=401, mimetype='text/plain', headers=[
-                ('WWW-Authenticate', 'Basic realm="Login Required"'),
-            ])
+            return Response(err_msg, status=401, mimetype='text/plain',
+                            headers=[('WWW-Authenticate', 'Basic realm="Login Required"')])
 
     request_args = get_request_args(request.args, params)
     on_demand = params.get('on_demand')
     created = datetime.now(timezone.utc)
-    edl = get_edl_on_demand() if on_demand else create_new_edl(request_args)
-    etag = f'"{hashlib.sha1(edl.encode()).hexdigest()}"'    # nosec
+    edl_data, original_indicators_count = get_edl_on_demand() if on_demand else create_new_edl(request_args)
     query_time = (datetime.now(timezone.utc) - created).total_seconds()
+    etag = f'"{hashlib.sha1(edl_data.encode()).hexdigest()}"'  # nosec
```

Comment:
please revert

Reply:
I didn't actually add this line as GitHub suggests, I just moved it down a line.

The code is:
```python
    created = datetime.now(timezone.utc)
    edl_data, original_indicators_count = get_edl_on_demand() if on_demand else create_new_edl(request_args)
    query_time = (datetime.now(timezone.utc) - created).total_seconds()
    etag = f'"{hashlib.sha1(edl_data.encode()).hexdigest()}"'  # nosec
    ```
    
Seems to make more sense to me that `query_time` is called right after calling the query, as it's a runtime counter.